### ISSUE TITLE
Coach reports summary main view issues

### ISSUE THREAD

**Issue by MCGallaspy** [learningequality/ka-lite#4349]
I'm collecting several different issues here. All of these are with data generated by running `generaterealdata --scenario-1` (see https://github.com/learningequality/ka-lite/pull/4347) on branch `0.15.x` (running from source in this case, not from this week's windows installer) and logged in as a coach user.

1. "Total Hours Logged" says is labeled "Hours spent on content vs total hours logged". Does "total hours logged" mean among all users for the Facility? The FacilityGroup? Among all facilities in the sharing network?
2. "Total Hours Logged" central value of "Total" is in hours (but its unlabeled), but the circle surrounding actually represents a ratio -- a bit misleading.
3. The central values in the main view are not vertically centered.
4. Some are just unadorned numbers, with no visualization as with "Total Hours Logged" -- is this intentional? See screenshot: 
![ss1](https://cloud.githubusercontent.com/assets/8888020/9641871/d14ca8c0-516d-11e5-8f1f-06a8fc176494.png)
5. Text overflows on mouseover of "Total Hours Logged": 
![ss2](https://cloud.githubusercontent.com/assets/8888020/9641895/0090ec86-516e-11e5-8a56-dd68291e73e6.png)
6. Selecting a new facility or group causes the main view to not be updated correctly -- on initially loading the page I see the screenshot from 4, but then choosing the "Ungrouped" group and then the original group again causes "Average progress in exercises" not to be updated: 
![ss3](https://cloud.githubusercontent.com/assets/8888020/9642030/c6967202-516e-11e5-93bb-c233832772c4.png)
(Edit: same thing in 6 happens if I switch facilities)
7. Clicking "Show Tabular Report" and then changing the facility or group causes "Hide Tabular Report" to be shown even though the tabular report disappears. Clicking it seems to have the same effect as "Show Tabular Report", so it's apparently just the text that's not updated: 
![ss3](https://cloud.githubusercontent.com/assets/8888020/9642152/633e43d2-516f-11e5-9d47-84f9eca838ae.png)

**Comment by MCGallaspy** [comment]
Edit: added point 7 above.

**Comment by rtibbles** [comment]
1. Can't change wording due to string freeze, might be able to clarify in docs.
2. Changed values to percentages.
3. Won't fix. The text changes size vertically when the text changes. Can be iterated on in 0.16.
4. Intentional, we merged what was ready.
5. Should be fixed by making font smaller.
6. Can't directly replicate, but selecting "Ungrouped" does seem to be doing something it ought not (it is getting all Ungrouped students, and not just those within a particular facility).
7. Odd, must need a rerender.
